README.md

```diff
@@ -32,12 +32,12 @@ Try to type something like `help` and let the magic happens! For a detail review
 
 ### Credits
 
-DiscordConsole authored by [Legolord208](https://github.com/legolord208) with help from [Tdbot](https://github.com/tbodt), [Mnpn](https://github.com/Mnpn03) and [Parallax](https://github.com/Doyouwanttomakeacake).
+DiscordConsole authored by [LEGOlord208](https://github.com/LEGOlord208), with documentation improvments by [Parallaχ](https://github.com/Doyouwanttomakeacake).
```

Comment:
> authored by
sounds odd...

Could you change it to `written by` or `coded by` or `made by` or whatever?
Again, SORRY for nitpicking!